spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/abstractnn/AbstractModule.scala

```diff
@@ -1062,6 +1067,54 @@ abstract class AbstractModule[A <: Activity: ClassTag, B <: Activity: ClassTag,
     (Array(scala.reflect.classTag[T]), Array(ev))
   }
 
+  // for mkl dnn model
+  val isMklDnnModel = false
```

Comment:
Should we create another abstract class MKLDNNAbstracModule to hold these new properties?